composer/datasets/in_context_learning_evaluation.py

```diff
@@ -153,10 +153,10 @@ def prep_examples(self, num_fewshot: int, prompt_string: str, example_delimiter:
 
             cont = f'{continuation_delimiter}{cont}'
 
-            encoded_example['context'] = self.tokenizer(ctxt)
-            encoded_example['continuation'] = self.tokenizer(cont)
             encoded_example['preamble'] = self.tokenizer(
                 preamble)  # if the preamble is empty then these will be 0-length lists
+            encoded_example['context'] = self.tokenizer(ctxt, add_special_tokens=len(preamble) == 0)
```

Comment:
I believe that when you tokenize an empty string, it still adds special tokens
```In [4]: tt = transformers.AutoTokenizer.from_pretrained('facebook/opt-125m')

In [5]: tt('')
Out[5]: {'input_ids': [2], 'attention_mask': [1]}```

So, your code still gets the right outcome I think, but it would be clearer to write it such that the `encoded_example['preamble']` is just an empty list if `preamble` is empty.